sa/model.go

```diff
@@ -534,7 +546,7 @@ func authzPBToModel(authz *corepb.Authorization) (*authz2Model, error) {
 				}
 			}
 		}
-		token, err := base64.StdEncoding.DecodeString(tokenStr)
+		token, err := base64.RawURLEncoding.DecodeString(tokenStr)
```

Comment:
Can you add some context for this change? Was `StdEncoding` incorrect?

Reply:
Yup, [tokens use `RawURLEncoding`](https://github.com/letsencrypt/boulder/blob/063a98f02a59590b6c7943d3812dcf5e09dbb9c7/core/util.go#L57-L70).